protocol-units/types/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "protocol-unit-types"
```

Comment:
As far as I understand @l-monninger's organization, `protocol-units` is a directory name used to group crates in the workspace, so it should not figure in the global-ish crate names.

This directory is also meant for protocol crates, but I'm not sure about common data definitions. There is also already `movement-types` fulfilling a similar role. Maybe we should coalesce the two as suggested in the other comment.

Reply:
To reduce the confusion, a description here, or maybe the top-level crate doc should describe the purpose of each crate.

Reply:
When reading the code, I would like to see a prefix on all crates indicating they are part of the movement ecosystem. I noted this to discuss in a later meeting. We should come up with a naming convention. I agree.